lxd/request/request.go

```diff
@@ -32,5 +32,8 @@ func CreateRequestor(r *http.Request) *api.EventLifecycleRequestor {
 	if ok {
 		requestor.Protocol = val
 	}
+
+	requestor.Address = r.RemoteAddr
```

Comment:
This line should go above in the `Normal requestor` block.

Then in this block you should add a chunk for `CtxForwardedAddress` too.